src/main/kotlin/lotto/domain/RankReward.kt

```diff
@@ -0,0 +1,47 @@
+package lotto.domain
+
+enum class RankReward(
+    val rankValue: Int,
+    val matchedCount: Int,
+    val money: Money,
+    val matchBonus: Boolean?,
+) {
+    FIRST(1, 6, Money(2_000_000_000), null),
+    SECOND(2, 5, Money(30_000_000), true),
+    THIRD(3, 5, Money(1_500_000), false),
```

Comment:
```suggestion
    val matchBonus: Boolean? = null,
) {
    FIRST(1, 6, Money(2_000_000_000)),
    SECOND(2, 5, Money(30_000_000), matchBonus = true),
    THIRD(3, 5, Money(1_500_000), matchBonus = false),
```

이렇게하면 각 값들을 볼때 조금 더 필요한 정보만을 볼 수 있을 것 같습니다!

Reply:
디폴트 파라미터를 이용해서 가독성을 올릴 수 있네요! 의견 감사합니다☺️